handout/features/spread_and_rest.md

```diff
@@ -33,10 +33,10 @@ let mapABC  = { a: 5, b: 6, c: 3};
 let mapABCD = { ...mapABC, d: 7};  // { a: 5, b: 6, c: 3, d: 7 }
 ```
 
-## Operator Rest
+## Rest parameter
 
-Rest arguments share the ellipsis like syntax of rest operators but are used
-for a different purpose. Rest arguments are used to access a variable number
+Rest parameters share the ellipsis like syntax of spread syntax but are used
+for a different purpose. Rest parameters are used to access indefinite number
```

Comment:
Why two different lines?

Also, proposed change: 

"Rest parameters look very similar to the "spread syntax" but are used for a different purpose. Rest parameters are used to access indefinite number of arguments passed to a function. For example:"